### ISSUE TITLE
Update Quick Tour for Removal of Welcome Page in 2.7

### ISSUE THREAD

**Issue by BT643** [symfony/symfony-docs#5353]
It seems the "Welcome!" page has been removed as of 2.7 so the quick tour documentation is now misleading / wrong.

https://symfony.com/doc/current/quick_tour/the_big_picture.html#running-symfony

It tells you to go to http://localhost:8000 to see the welcome page but this just brings up an error as there is now no route defined for /.

I was going to update the documentation but wasn't sure what to put there instead? Just tell the user to go straight to /app/example and see the "Homepage." text I guess would be the best option?

**Comment by michaelperrin** [comment]
This can be indeed misleading for new comers. The whole Quick Tour should be rewritten with either a simple step by step new app (creating the controller / routing / template) or with the new demo app (http://symfony.com/blog/introducing-the-symfony-demo-application).

I would prefer the first solution, as Symfony is installed by developers who want to start a new app and see how things work.

**Comment by cuthbert19** [comment]
I am indeed a newcomer and I was at first very disappointed, while following the quick tour, of not being able to get it to work right after installation. I second michaelperrin proposal, the demo application seems to me as being too complex for a quick tour, while I appreciated the "10 minutes by 10 minutes" lesson of the current approach.

**Comment by WouterJ** [comment]
https://github.com/symfony/symfony-docs/pull/5416 is now merged, which fixes the URL. The screenshot has already been updated in another PR. This means that this issue is now completely fixed.

Rewriting the complete Quick Tour, now we got a demo application, is a very good question. We were discussing this internally, but I don't know about the current state of this. Anyway, it's better to have a dedicated issue for this :)

**Comment by xabbuh** [comment]
It looks like the changes from #5400 and #5416 are not sufficient enough. Firstly, the changes were only merged in the `2.7` branch while the Symfony Standard Edition has also been updated for older maintained versions. Secondly, there were some other wrong URL and text references. I proposed updates for these things in #5461.

**Comment by BT643** [comment]
There's an issue opened on StackOverflow as little as 9 hours ago which show this is still an "issue" also:

http://stackoverflow.com/questions/31227664/installing-symfony-2-on-windows

**Comment by xabbuh** [comment]
@BT643 These issue would be tackled by #5461, right?

**Comment by BT643** [comment]
@xabbuh It would certainly help, although I imagine there would still be some people who would start the built in server from the console which tells you to visit / and wonder why they're getting an "error page". 

If that is all properly explained in the quick tour it should resolve it.

**Comment by xabbuh** [comment]
@BT643 Do you have an idea how we could make this more clear?

**Comment by BT643** [comment]
When you run the command: `symfony new testing123` you get:

    * Run your application:
            1. Execute the php app/console server:run command.
            2. Browse to the http://localhost:8000 URL.

People may follow this and immediately wonder why it's not working. Yes, they will see the following if they carry on reading, but may still wonder why / is not working as it isn't explained.

> Open your browser and access the http://localhost:8000/app/example URL to see the welcome page of Symfony:

A possible solution would just be to modify the note below it slightly:

> Instead of the welcome page, you may see a blank page or an error page. This is caused by a directory permission misconfiguration. There are several possible solutions depending on your operating system. All of them are explained in the Setting up Permissions section of the official book. **Please note - You will receive a 404 if you visit http://localhost:8000/ as no route has been defined here yet.**

(the bold section is just to show what I changed, rather than it being bold in the docs)

Obviously this is all blindingly obvious when you already know it, but I'm just looking from a brand new user's perspective.

**Comment by javiereguiluz** [comment]
I agree with @BT643. I consider this a "bug" and a high priority one. The first thing you see about Symfony shouldn't be an error. This is WTF for experienced developers and totally unacceptable for newcomers.

Sadly I don't know what to do. What could we do to improve user experience here? I need ideas. Thanks! / cc @WouterJ @xabbuh @weaverryan

**Comment by xabbuh** [comment]
Maybe we should update the Standard Edition to include a redirect from `/` to `/app/example`? I am not sure if there is something that we could write in the docs to make this more clear (and actually having an application that provides things under `/app/example`, but returns a 404 error at `/` looks weird to me).

**Comment by javiereguiluz** [comment]
In an ideal world, when you access the `/` URL of a freshly installed Symfony project, you would see a simple but beautiful Welcome Page. This page shouldn't pollute your application, so it shouldn't be generated with a custom route or controller or template. Is this even possible? Can we make this check in the exception listener of the `dev` environment and display the Welcome page if there is no custom route defined for that URL?

**Comment by xabbuh** [comment]
Technically that would be possible. Though I am not sure if it really fits best there.

**Comment by weaverryan** [comment]
I actually just messaged @javiereguiluz about this. I'm strongly in favor of a route+controller+template at the `/` URL of the standard edition. I don't think this pollutes the application at all. To the contrary, beginners will have a starting point they can start tweaking. For experienced devs, it's just one controller and one templates.

I merged #5461 so the quick tour is up to date with how things are *now*. Hopefully we can update that again soon to show some elegant "homepage".
